src/index.js

```diff
@@ -1,3 +1,4 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
You do not need to import thins bootstrap since everything is already confgured. Just pull the latest code and work on it.